lib/realtime/server.ex

```diff
@@ -345,15 +344,13 @@ defmodule Realtime.Server do
     {:noreply, %{state | active_alert_route_ids: new_active_route_ids}}
   end
 
-  defp update_vehicle_positions(
+  defp remove_inactive_keys(
```

Comment:
**praise:** I like having this separated out into a separate function from the rest of the update logic.